src/client/components/informationPanel/BasicInfo.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Text, theme, Button } from '@aragon/ui'
+import { Social, Link, IconLocation, IconGlobe } from '../styled-components'
+
+export const Name = ({ name, handleOpenEdit }) => {
+  return name ? (
+    <Text.Block size="xxlarge" css="font-weight: 700">
+      {name}
+    </Text.Block>
+  ) : (
+    <Center>
+      <Button onClick={handleOpenEdit}>
+        <Text size="large" color={theme.accent}>
+          Add name
+        </Text>
+      </Button>
+    </Center>
+  )
+}
+
+Name.propTypes = {
+  name: PropTypes.string,
+  handleOpenEdit: PropTypes.func.isRequired,
+}
+
+export const Description = ({ description, handleOpenEdit }) =>
+  description ? (
+    <Text.Block>{description}</Text.Block>
+  ) : (
+    <Center>
+      <Button size="small" onClick={handleOpenEdit}>
+        <Text size="large" color={theme.accent}>
+          Add bio
+        </Text>
+      </Button>
+    </Center>
+  )
+
+Description.propTypes = {
+  description: PropTypes.string,
+  handleOpenEdit: PropTypes.func.isRequired,
+}
+
+export const Location = ({ location, handleOpenEdit }) => (
+  <Social>
+    <IconLocation width="13px" height="13px" color={theme.textTertiary} />
+    {location ? (
+      <Text size="normal" color={theme.textTertiary}>
+        {location}
+      </Text>
+    ) : (
+      <Button compact size="small" onClick={handleOpenEdit} css="width: 80px">
```

Comment:
`<Social>` expects two children, an icon and then the content which takes up the rest of the available width. Put this `<Button>` in a div and everything should look ok!

Reply:
Thank you, it works. I decreased padding on the button to make it look like before:
![Screenshot from 2019-07-03 23-33-44](https://user-images.githubusercontent.com/34452131/60627056-1e240d80-9ded-11e9-9e91-d02d43170cd0.png)
Not sure if it's ok though - add a few pixels perhaps?

Reply:
I added a new commit that fixes this. Thanks for the helpful screenshot!